lib/watcher.js

```diff
@@ -126,6 +128,7 @@ class Watcher {
 				.then(runStatus => {
 					runStatus.previousFailCount = this.sumPreviousFailures(currentVector);
 					logger.finish(runStatus);
+					logger.write(rerunMessage);
```

Comment:
Unfortunately this doesn't work with the mini reporter. I think `finish()` should take an optional second parameter, call it `footer` perhaps, and both the mini and verbose loggers can then include this in their regular output.